lib/features/digital_guide_view/general_info/presentation/digital_guide_view.dart

```diff
@@ -73,13 +73,14 @@ class _DigitalGuideView extends ConsumerWidget {
                 .replaceAll("ulica", "ul."),
             icon: Assets.svg.contactIcons.compass,
           ),
-          ContactIconsModel(
-            text: digitalGuideResponseExtended.telephoneNumber,
-            icon: Assets.svg.contactIcons.phone,
-            // TODO(Bartosh): url not working, nothing happens
-            url:
-                "tel:+48${digitalGuideResponseExtended.telephoneNumber.replaceAll("<p>", "").replaceAll("</p>", "")}",
-          ),
+          if (digitalGuideResponseExtended.telephoneNumber.isNotEmpty)
+            ContactIconsModel(
+              text: digitalGuideResponseExtended.telephoneNumber,
+              icon: Assets.svg.contactIcons.phone,
+              // TODO(Bartosh): url not working, nothing happens
+              url:
+                  "tel:+48${digitalGuideResponseExtended.telephoneNumber.replaceAll("<p>", "").replaceAll("</p>", "")}",
```

Comment:
Is this URL working?

Reply:
imo we should def move the logic that extracts relevant data to repo layer and prob make it as an extension

Reply:
It was not in the scope of this task but I might try implementing it. WDYT? But I think that the existing extension for URLs might work and I don't think that another one is necessary

Reply:
I mean extension for replacing the <p> in the string, but u right, an overkill here. If you can please make it works and we re ready to merge then